### ISSUE TITLE
Access to originally defined external terms

### ISSUE THREAD

**Issue by sidvishnoi** [tabatkins/bikeshed#1705]
When Bikeshed updates anchor data from Shepherd, it [lower cases](https://github.com/tabatkins/bikeshed/blob/b76f9052e04d33f3db06c129524d656843a82f35/bikeshed/update/updateCrossRefs.py#L283-L284) [`lowercaseTypes`](https://github.com/tabatkins/bikeshed/blob/e4ffdf22d9286f77741537adec50ff66905fbf23/bikeshed/config/dfnTypes.py#L62), which is a good thing.
While searching for terms like `sRGB`, `UTF-8`, `default toJSON steps` etc, it would be nice if there was a way to access the originally defined term, and not just the lowercase ones. This would benefit fuzzy searching at https://respec.org/xref as well as `bikeshed refs`.

In short, consider adding a `originalTerm` to https://github.com/tabatkins/bikeshed/blob/b76f9052e04d33f3db06c129524d656843a82f35/bikeshed/update/updateCrossRefs.py#L335-L344

cc: @marcoscaceres

**Comment by marcoscaceres** [comment]
This would be really helpful for when suggesting "how to cite" terms... right now, for example, we are suggesting "[=srgb=]" for sRGB, which is not great :( same for the others.
